modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -910,7 +910,7 @@ private ObjectDefinition _toObjectDefinition(
 				actions = HashMapBuilder.put(
```

Comment:
Change commit to `Modifiable system object definitions can only be deleted by allowed bundles`